test/passing/refs.ocamlformat/source.ml.ref

```diff
@@ -9435,19 +9435,19 @@ let h = fun ?l:(p = 1) ?y:u ?(x = 3) -> 2
 
 let _ = function
   | a, s, ba1, ba2, ba3, bg -> begin
-    ignore
-      ( Array.get x 1 + Array.get [||] 0 + Array.get [|1|] 1
-      + Array.get [|1; 2|] 2 ) ;
-    ignore [String.get s 1; String.get "" 2; String.get "123" 3] ;
-    ignore (ba1.{0} + ba2.{1, 2} + ba3.{3, 4, 5}) ignore bg.{1, 2, 3, 4}
+      ignore
+        ( Array.get x 1 + Array.get [||] 0 + Array.get [|1|] 1
+        + Array.get [|1; 2|] 2 ) ;
+      ignore [String.get s 1; String.get "" 2; String.get "123" 3] ;
+      ignore (ba1.{0} + ba2.{1, 2} + ba3.{3, 4, 5}) ignore bg.{1, 2, 3, 4}
     end
```

Comment:
Just to note that this is a very large change, it results in +1163 -1141 diffs in `test_branch`. I'd consider this to be a bug fix. Is here any user of the `ocamlformat` profile ?

Reply:
I use it in my projects, and I would love having this bugfix.

Maybe changes entry is warranted ?